tests/snapshot_tests/snapshot_apps/border_alpha.py

```diff
@@ -1,43 +1,40 @@
 from textual.app import App, ComposeResult
-from textual.containers import Grid
-from textual.widget import Widget
+from textual.containers import Vertical
```

Comment:
Rewrote this snapshot test, because it wasn't clear to me what it should look like.

Reply:
The problem I see with the rewrite is that it doesn't exercise the CSS parser in quite the same way. The original test was designed to try out a good combination of styles, colours and alpha values to ensure it all worked together.

That looks to have been lost.

Reply:
True. But those are better captured with a unit test. Snapshots should be used to validate that things look correct only, where you can't tell from the data alone.